docs/docs/api/promise.coroutine.md

```diff
@@ -13,7 +13,9 @@ title: Promise.coroutine
 Promise.coroutine(GeneratorFunction(...arguments) generatorFunction, Object options) -> function
 ```
 
-Returns a function that can use `yield` to yield promises. Control is returned back to the generator when the yielded promise settles. This can lead to less verbose code when doing lots of sequential async calls with minimal processing in between. Requires node.js 0.12+, io.js 1.0+ or Google Chrome 40+.
+In a typical use case, the parameter `generatorFunction` should be a `GeneratorFunction` producing a generator that yields one or more `Promise`s. (Also see the Tips section below.)
```

Comment:
Just to be clear, [a generator is not a `GeneratorFunction`, and is an "iterator"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Generator) :

>  The Generator object is returned by a generator function and it conforms to both the iterable protocol and the iterator protocol.